### DISCUSSION TITLE
Getting error 404: page not found

### DISCUSSION THREAD

**Discussion by syrup-panini-bit** [transitland/transitland#444] [Q&A] [answered]
I'm trying to call the API with this: https://transit.land/api/v2/rest/api/v2/rest/agencies?agency_key=o-9q5-metro~losangeles, and I'm always getting "404 page not found." What am I doing wrong?

**Comment by drewda** [comment, marked as answer]
Your URL has some duplication in it.

Try `https://transit.land/api/v2/rest/agencies?agency_key=o-9q5-metro~losangeles` (plus your API key)
